src/plugins/orangehrmClaimPlugin/Api/Model/ClaimRequestModel.php

```diff
@@ -47,7 +70,8 @@ public function __construct(ClaimRequest $claimRequest)
                 'referenceId',
                 ['claimEvent', 'id'],
                 ['claimEvent', 'name'],
-                'currency',
+                ['currencyType', 'id'],
+                ['currencyType', 'name'],
```

Comment:
hope this is align with `src/plugins/orangehrmPimPlugin/Api/Model/EmployeeSalaryModel.php`

Reply:
aligned with it. response come like this.
```
"currencyType": {
            "id": "USD",
            "name": "United States Dollar"
        },
```